auth-web/tests/unit/test-utils/test-data/affiliations.ts

```diff
@@ -86,10 +105,18 @@ export const businesses = [
     goodStanding: false,
     businessIdentifier: 'TIQcIs5sss',
     corpType: {
-      code: 'TMP'
+      code: CorpTypes.INCORPORATION_APPLICATION
     },
     corpSubType: {
-      code: 'BEN'
+      code: CorpTypes.BENEFIT_COMPANY
     }
   }
 ]
+
+function toDateUTC (dateString) {
+  const date = new Date(dateString)
+  const utcDate = new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate(),
```

Comment:
Use moment or one of the date libraries already installed. No point in reinventing the wheel.